device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/IoTHubConnectionType.java

```diff
@@ -7,5 +7,6 @@
 {
     UNKNOWN,
     SINGLE_CLIENT,
-    USE_TRANSPORTCLIENT
+    USE_TRANSPORTCLIENT,
```

Comment:
Any comment here to indicate the transport client is being deprecated?

Reply:
There isn't a way to formally deprecate an enum value, but I can certainly add a comment for our understanding

Reply:
I can deprecate it and add a comment about why, sure